ci-operator/step-registry/single-node/conf/e2e/graceful-shutdown/single-node-conf-e2e-graceful-shutdown-ref.metadata.json

```diff
@@ -0,0 +1,16 @@
+{
+	"path": "single-node/conf/e2e/graceful-shutdown/single-node-conf-e2e-graceful-shutdown-ref.yaml",
+	"owners": {
+		"approvers": [
+			"romfreiman",
+			"eranco74",
+			"tsorya",
+			"osherdp",
+			"yuvigold",
+			"omertuc",
+			"wgahnagl",
+			"rphillips",
+			"mrunalp"
+		]
+	}
+}
```

Comment:
same as above - i think this just happend on your last push?

Reply:
this is also autogenerated :ok_hand: